src/osdc/CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
 set(osdc_files
   Filer.cc
   ObjectCacher.cc
-  Objecter.cc
-  error_code.cc
-  Striper.cc)
+  error_code.cc)
```

Comment:
Please add a comment. That's pretty confusing.

Reply:
I don't understand - what confuses you? That some source files in this directory are missing? Others are missing, too, but there is no comment.

Reply:
> That some source files in this directory are missing?

Yes

> Others are missing, too, but there is no comment.

Extra confusing!

Reply:
OK, I added a comment